### DISCUSSION TITLE
FW-Update von Fusion 2.1 DTU nicht möglich

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2064] [Q&A]
### What happened?

Hallo,

ich habe seit einigen Wochen eine Fusion 2.1 DTU mit der FW v24.5.6, mit der ich einen HMS-1000 monitore.
Nun wollte ich die DTU auf die aktuelle FW v24.6.10 updaten, aber alle Versuche schlagen fehl mit der Meldunng:

![DTU-FW-Problem_2024-06-14](https://github.com/tbnobody/OpenDTU/assets/140426500/2d4a25ed-e341-4322-8977-f9592a3cf8e5)

Auch Ein- und Ausschalten der DTU bringen dasselbe Ergebnis.
Kann mir hier jemand bitte helfen? Ich habe aktuell keine Idee mehr, bin aber auch nicht so der SW-Profi.

Ich habe noch eine andere DTU (war früher mal an einem HM-800), die hat sich - wie immer schon - ohne Probleme updaten lassen.

Vielen Dank im Voraus und Grüße
Werner

### To Reproduce Bug

FW-Update über DTU starten

### Expected Behavior

FW wird aktualisiert auf DTU

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

v24.5.6,

### Relevant log/trace output

```Shell
nein
```

### Anything else?

Danke für Hilfe

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of OpenDTU, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by tbnobody** [comment]
I am going to migrate this issue to a discussion because I don't think it affects all users and it was already mentioned in serveral in many issues.

**Comment by tbnobody** [comment]
Stelle doch mal bitte einen Screenshot Info --> System --> Speicherinformationen ein.

Ich denke dein Problem liegt hieran: https://www.opendtu.solar/firmware/howto/upgrade_partition/

**Comment by Fokker-1** [comment]
Hallo,

danke für das Verschieben hierhin.

Ich hatte zwischenzeitlich Mail-Kontakt mit dem Geschäft (3Print-D Solution), bei dem ich meine Fusion DTU gekauft hatte und dort auch das Problem geschildert.

Es konnte dann schnell gelöst werden. Für diese Fusion DTU muss die folgende FW geflasht werden (OTA):

https://github.com/tbnobody/OpenDTU/releases/download/v24.6.10/opendtu-generic_esp32s3_usb.bin

Jetzt ist alles ok,, die aktuelle FW ist nun drauf! Man muss "nur" wissen, was genau man machen muss ...
Mein Dank an 3Print-D Solution für die schnelle und kompetente Reaktion.

Gruß
Werner

**Reply by jachen** [reply]
Danke. Stunden danach gesucht.

**Reply by DPO99** [reply]
> Hallo,
> 
> danke für das Verschieben hierhin.
> 
> Ich hatte zwischenzeitlich Mail-Kontakt mit dem Geschäft (3Print-D Solution), bei dem ich meine Fusion DTU gekauft hatte und dort auch das Problem geschildert.
> 
> Es konnte dann schnell gelöst werden. Für diese Fusion DTU muss die folgende FW geflasht werden (OTA):
> 
> https://github.com/tbnobody/OpenDTU/releases/download/v24.6.10/opendtu-generic_esp32s3_usb.bin
> 
> Jetzt ist alles ok,, die aktuelle FW ist nun drauf! Man muss "nur" wissen, was genau man machen muss ... Mein Dank an 3Print-D Solution für die schnelle und kompetente Reaktion.
> 
> Gruß Werner

Sowas steht übrigens auch in den jew. Docus. Hier z.B. https://github.com/markusdd/OpenDTUFusionDocs/blob/main/OTA.md

**Comment by Fokker-1** [comment]
.... gerne, prima, dass es geklappt hat!
